### PR TITLE
snoop: add French translation

### PR DESCRIPTION
**Pull request by jay-tau**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/sunos/snoop.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# snoop
+
+> Renifleur de paquets réseau.
+> Équivalent SunOS de tcpdump.
+> Plus d'information: <https://www.unix.com/man-page/sunos/1m/snoop>.
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:5]:
```suggestion
> Plus d'information : <https://www.unix.com/man-page/sunos/1m/snoop>.
```

File: pages.fr/sunos/snoop.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Capturer des paquets sur une interface réseau spécifique:
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:7]:
```suggestion
- Capturer des paquets sur une interface réseau spécifique :
```

File: pages.fr/sunos/snoop.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -d {{e1000g0}}`
+
+- Enregistrer les paquets capturés dans un fichier au lieu de les afficher:
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:11]:
```suggestion
- Enregistrer les paquets capturés dans un fichier au lieu de les afficher :
```

File: pages.fr/sunos/snoop.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -o {{filename}}`
+
+- Afficher le résumé détaillé de la couche de protocole des paquets d'un fichier:
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:15]:
```suggestion
- Afficher le résumé détaillé de la couche de protocole des paquets d'un fichier :
```

File: pages.fr/sunos/snoop.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -V -i {{filename}}`
+
+- Capturez les paquets réseau provenant d'un nom d'hôte et accédez à un port donné:
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:19]:
```suggestion
- Capturez les paquets réseau provenant d'un nom d'hôte et accédez à un port donné :
```

File: pages.fr/sunos/snoop.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop to port {{port}} from host {{hostname}}`
+
+- Capturez et affichez un vidage hexadécimal des paquets réseau échangés entre deux adresses IP:
```
Comment (nicokosi) [inline pages.fr/sunos/snoop.md:23]:
```suggestion
- Capturez et affichez un vidage hexadécimal des paquets réseau échangés entre deux adresses IP :
```

File: pages.fr/sunos/snoop.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -o {{filename}}`
+
+- Afficher le résumé détaillé de la couche de protocole des paquets d'un fichier :
+
+`snoop -V -i {{filename}}`
+
+- Capturez les paquets réseau provenant d'un nom d'hôte et accédez à un port donné :
+
+`snoop to port {{port}} from host {{hostname}}`
```
Comment (navarroaxel) [inline pages.fr/sunos/snoop.md:21]: Please translate the placeholders, like `hostname`, `port`, `id_address`, etc., for get this easier for non-English speakers.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**